modules/persistent-storage-csi-tp-enable.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+// persistent-storage-csi-azure.adoc
+
+ifeval::["{context}" == "persistent-storage-csi-vsphere"]
+:vsphere:
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-azure"]
+:azure:
+endif::[]
+
+[id="csi-tp-enable_{context}"]
+= Enabling the {FeatureName} CSI driver operator
+To enable the {FeatureName} Container Storage Interface (CSI) driver operator, you must enable feature gates with the `TechPreviewNoUpgrade` feature set.
+
+.Procedure
+
+. Enable feature gates with the `TechPreviewNoUpgrade` feature set (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#[Enabling features using feature gates]).
```

Comment:
You'll need to remove the xref as they can't be included in modules yet. Suggest something like, "See "Enabling features using feature gates" and then adding an xref in the assembly file under "Additional resources".